src/it/scala/OmegaRepositoryISpec.scala

```diff
@@ -89,14 +89,34 @@ class OmegaRepositoryISpec
         result.success.get.length mustBe 2
       }
     }
-    "must return a List containing the HHC.1 agent" in {
+    "must return a List containing with all HHC agents created from a date onwards (0)" in {
       when(mockConfig.sparqlEndpoint).thenReturn("http://localhost:8080/rdf4j-server/repositories/PACT")
       eventually {
         val result = repository.getAgentDescriptionEntities(
-          ListAgentSummary(versionTimestamp = Some("2023-01-25T14:14:47.534Z")),
+          ListAgentSummary(versionTimestamp = Some("2023-08-01T00:00:00.000Z")),
           new URI("http://cat.nationalarchives.gov.uk/agent.HHC")
         )
-        result.success.get.head.identifier mustBe "http://cat.nationalarchives.gov.uk/agent.HHC.1"
+        result.success.get mustEqual List.empty
+      }
+    }
+    "must return a List containing with all HHC agents created from a date onwards (1)" in {
+      when(mockConfig.sparqlEndpoint).thenReturn("http://localhost:8080/rdf4j-server/repositories/PACT")
+      eventually {
+        val result = repository.getAgentDescriptionEntities(
+          ListAgentSummary(versionTimestamp = Some("2023-07-01T00:00:00.000Z")),
+          new URI("http://cat.nationalarchives.gov.uk/agent.HHC")
```

Comment:
I wonder if we should put `http://cat.nationalarchives.gov.uk` in a constant somewhere - perhaps the in the package object? and/or maybe even create some URI builder functions, such as:

```scala
val catNS = "http://cat.nationalarchives.gov.uk/"
val agentUriPrefix = catNS + "agent."

def agentUri(id: Int): URI = agentUri(encode(id))

def agentUri(id: String): URI = {
   new URI(agentUriPrefix + "." + id)
}
```

Reply:
In fact I already created a `BaseURL` object, just wasn't using it everywhere. I have now updated the PR.